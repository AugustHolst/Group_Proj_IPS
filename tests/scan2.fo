fun int incr(int a, int b) = a + b

fun int writeInt(int n) = write(n)

fun [int] main() =
  let a = {0-4, 0, 29834928, 0-1387428} in
  let b = scan(incr, 0, a) in
  map(writeInt, b)
