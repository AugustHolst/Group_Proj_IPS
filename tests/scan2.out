-4 -4 29834924 28447496 